src/java/org/apache/cassandra/service/accord/AccordSafeCommand.java

```diff
@@ -127,9 +129,12 @@ public Command original()
         return original;
     }
 
-    public SavedCommand.SavedDiff diff()
+    public SavedCommand.Writer<TxnId> diff()
     {
-        return SavedCommand.diff(original, current);
+        if (current == null)
```

Comment:
doesn't this mean `preExecute` wasn't called?  What about when the status is `Uninitialised` (aka loaded a cmd we don't know)?  Some code tries to filter that, but this doesn't?